code/index.html

```diff
@@ -2,21 +2,137 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dentnews</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <header class="header">
+      <div class="hamburger"> 
+        <a href="./hamburger.html">
+          <!--divs to create three lines for hamburger menu-->
+          <div></div>
+          <div></div>
+          <div></div>
+        </a>
+      </div>
+      <img class="logo-1" src="./photos/dentnews.png" alt="Dentnews logo">
+      <nav class="navigation-container">
```

Comment:
Let's remember about correct indentations - right now it's not very descriptive and easy to read code!

Reply:
I’m not exactly sure what you want me to change in this part.

I have used the quick command shift + alt + F to format the code in Visual Studio and it gives me those indentations. 
  <header>
+  <header class="header">
+    <div class="hamburger"> 
+      <a href="./hamburger.html">
+      <!--divs to create three lines for hamburger menu-->
+      <div></div>
+      <div></div>
+      <div></div>
+      </a>
+    </div>
+    <img class="logo-1" src="./photos/dentnews.png" alt="Dentnews logo">
+    <nav class="navigation-container">

Does this look any better?

Reply:
```
<header>
  <header class="header">
    <div class="hamburger"> 
      <a href="./hamburger.html">
      <!--divs to create three lines for hamburger menu-->
      <div></div>
      <div></div>
      <div></div>
      </a>
    </div>
    <img class="logo-1" src="./photos/dentnews.png" alt="Dentnews logo">
    <nav class="navigation-container">
```

Reply:
Thank you for the snippet! When I placed comment on line 20, I meant to tell you about problems with code below (lines 21-34). Notice that there is not indentations indicating that all of those divs are nested inside nav. They look like if they were siblings